bump-version.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+
+import sys
+import subprocess
+
+def bump_version(version, level):
+    semver = {'major': 0, 'minor': 1, 'patch': 2}
+    version_pieces = version.lstrip('v').split('.')
+    version_pieces[semver[level]] = str(int(version_pieces[semver[level]]) + 1)
+
+    return "v" + ".".join(version_pieces)
+
+version = subprocess.run("git tag | sort -V | tail -n 1", shell=True, capture_output=True) \
```

Comment:
nit: I try to write code without the line continuations wherever possible, just because I think they're ugly. I probably would have written this as some variant of this:

```python
version = subprocess.run(
    "git tag | sort -V | tail -n 1", shell=True, capture_output=True
).stdout.decode().strip()
```

No need to change this, just letting you know my personal style.

Reply:
Whoops, we did do that in another file, my bad!